src/handler.ts

```diff
@@ -11,11 +12,18 @@ console.log(
   `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_PROVIDER_STAGE}'\n region: '${AWS_PROVIDER_REGION}'\n\n`,
 );
 
-const handler = async (event: APIGatewayEvent): Promise<{ statusCode: number; body: string }> => {
-  console.log(`Function triggered by event: ${event.httpMethod}`);
-
+const handler = async (event: any): Promise<{ statusCode: number; body: string }> => {
```

Comment:
Shouldn't be regressing to `any`. Create a custom type if you need to.

Reply:
I think you can mark it as optional 
`APIGatewayEvent?`